### PR TITLE
Docs/faq: Initial FAQ page

### PR DESCRIPTION
**Pull request by vzpgb**

### CODE REVIEW

File: content/resources/faq/_index.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+Your browser can not verify this certificate. Once you install a trusted SSL
+certificate instead of generated by our script, this warning will go away.
+
+
+**I want to run Compose Services on the server, how to configure
+ it.**
+
+To configure Compose Services with your domain, please change the `BASE_URL`
+field in the `fence_config.yaml` file, and the `hostname` field in the
+`peregrine.json`, `sheepdog.json` files.
```
Comment (hugheslaron) [inline content/resources/faq/_index.md]: To configure Compose Services with your domain, please change the `BASE_URL`
field in the `fence_config.yaml` file, and the `hostname` field in the
`peregrine.json`and `sheepdog.json` files.

Comment (vzpgb) [reply content/resources/faq/_index.md]: Thanks! Updated in 7990669

File: content/resources/faq/_index.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+file](https://github.com/uc-cdis/data-portal/blob/master/data/config/default.json)
+as a template for your own `gitops.json` file. Set navigation title and login
+title at your choice. Then reroll portal with `gen3 kube-setup-portal`.
+
+
+**I updated Fence configuration but changes don't appear?**
+
+After updating the Fence configuration, run `kubectl delete secret
+fence-config`, `gen3 kube-setup-fence` and roll Fence for configuration changes
+to take effect with `gen3 roll fence`
```
Comment (hugheslaron) [inline content/resources/faq/_index.md]: `gen3 roll fence`.

Comment (hugheslaron) [reply content/resources/faq/_index.md]: added a period

File: content/resources/faq/_index.md (hunk 1 above)
Comment (hugheslaron) [reply content/resources/faq/_index.md]: replace , with and

File: content/resources/faq/_index.md [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+
+**I'm storing `user.yaml` in S3 (not locally) and updated it, how to ensure
+changes take effect?**
+
+After you change `user.yaml`, run `gen3 kube-setup-secrets` and `gen3 job run
+usersync`.
+
+
+**I see my data on a Query page, but not in Exploration page. What other
+services do I need to deploy/configure to have it running?**
```
Comment (hugheslaron) [inline content/resources/faq/_index.md]: but not in the Exploration page.

Comment (hugheslaron) [reply content/resources/faq/_index.md]: added 'the'

Comment (vzpgb) [reply content/resources/faq/_index.md]: Thank you! Fixed in b7bca75

File: content/resources/faq/_index.md (hunk 2 above)
Comment (vzpgb) [reply content/resources/faq/_index.md]: Thank you! Fixed in b7bca75

**Pull request merged**